examples/example-base/test-base.cpp

```diff
@@ -0,0 +1,58 @@
+#include "test-base.h"
+
+#ifdef _WIN32
+#include <windows.h>
+#include <shellapi.h>
+int TestBase::parseOption(int argc, char** argv)
+{
+    wchar_t** szArglist;
+    int nArgs;
+    int i;
+
+    szArglist = CommandLineToArgvW(GetCommandLineW(), &nArgs);
+    for (i = 0; i < nArgs; i++)
+    {
+        if (wcscmp(szArglist[i], L"--test-mode") == 0)
+        {
+            m_isTestMode = true;
+        }
+    }
+
+    return 0;
+}
+#else
+int TestBase::parseOption(int argc, char** argv)
```

Comment:
I guess it will be fine if we just use these version.

Reply:
act